lib/Draggable.es6

```diff
@@ -319,10 +319,14 @@ export default class Draggable extends React.Component {
       style = createCSSTransform(transformOpts);
     }
 
+    const childProps = this.props.children.props
+
     // Mark with class while dragging
-    const className = classNames((this.props.children.props.className || ''), 'react-draggable', {
+    const className = classNames(childProps.className, 'react-draggable', {
```

Comment:
How about we make `classNameDragging` and `classNameDragged`'s value in `defaultProps` equal to `react-draggable-dragging` and `react-draggable-dragged`.